source/wp-content/themes/wporg-showcase-2022/src/style/style.scss

```diff
@@ -142,3 +142,7 @@
 		}
 	}
 }
+
+.search-no-results .wporg-archive-results-bar {
+	display: none;
+}
```

Comment:
I couldn't find a way to conditionally render this if there were sites found. There is a `<!-- wp:query-no-results -->` block, but not a `<!-- wp:query-results -->` block, and the layout is too different between the two. Hiding the top one based on the body class is the best I can come up with. Any better solution?

Results:
![Screen Shot 2022-12-07 at 12 56 24 PM](https://user-images.githubusercontent.com/1017872/206051434-469d92ac-c347-4294-89ca-af8499b69bdf.jpg)

No results:
![Screen Shot 2022-12-07 at 12 56 03 PM](https://user-images.githubusercontent.com/1017872/206051451-97dfdc04-d7e3-4738-9b30-ed36a79b59d8.jpg)